plugins/shell/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+<meta charset="utf-8" />
+<link rel="shortcut icon" href="favicon.png" />
+<title>Shell ▲ Prism plugins</title>
+<base href="../.." />
+<link rel="stylesheet" href="style.css" />
+<link rel="stylesheet" href="themes/prism.css" data-noprefix />
+<link rel="stylesheet" href="plugins/shell/prism-shell.css" data-noprefix />
+<script src="prefixfree.min.js"></script>
+
+<script>var _gaq = [['_setAccount', 'UA-33746269-1'], ['_trackPageview']];</script>
+<script src="http://www.google-analytics.com/ga.js" async></script>
+</head>
+<body>
+
+<header>
+	<div class="intro" data-src="templates/header-plugins.html" data-type="text/html"></div>
+
+	<h2>Shell</h2>
+	<p>Display a command prompt and output for a shell session.</p>
+</header>
+
+<section class="language-markup">
+	<h1>How to use</h1>
+
+	<p>Obviously, this is intended for code/kbd blocks (<code>&lt;pre>&lt;kbd></code>) and not for inline code.</p>
+
+	<p>Add class <code>shell</code> to your <code>&lt;pre></code> and specify the user and host names to display in the prompt using the <code>data-user</code> and <code>data-host</code> attributes.</p>
+
+	<p>Optional: You may specify the lines to be presented as output (no prompt) through the <code>data-output</code> attribute on the <code>&lt;pre></code> element in the following simple format:</p>
+	<ul>
+		<li>A single number refers to the line with that number</li>
+		<li>Ranges are denoted by two numbers, separated with a hyphen (-)</li>
+		<li>Multiple line numbers or ranges are separated by commas.</li>
+		<li>Whitespace is allowed anywhere and will be stripped off.</li>
+	</ul>
+
+	<p>Examples:</p>
+	<dl>
+		<dt>5</dt>
+		<dd>The 5th line</dd>
+
+		<dt>1-5</dt>
+		<dd>Lines 1 through 5</dd>
+
+		<dt>1,4</dt>
+		<dd>Line 1 and line 4</dd>
+
+		<dt>1-2, 5, 9-20</dt>
+		<dd>Lines 1 through 2, line 5, lines 9 through 20</dd>
+	</dl>
+</section>
+
+<section>
+	<h1>Examples</h1>
+
+	<h2>Root User Without Output</h2>
+<pre class="shell" data-user="root" data-host="localhost"><kbd class="language-shell">cd /usr/local/etc
+cp php.ini php.ini.bak
+vi php.ini</kbd></pre>
+
+	<h2>Non-Root User With Output</h2>
+<pre class="shell" data-user="chris" data-host="remotehost" data-output="2, 4-8"><kbd class="language-shell">pwd
```

Comment:
It works on your blog, because you use a general CSS reset in `all.min.css`. The CSS reset sets the font of the `kbd` tag to `font: inherit`.

I've just pulled the newest version and tried again with Chromium and Firefox and it still doesn't work. It probably works for you, because you have other fonts installed than me. Prism sets the `font-family` to:

```css
code[class*="language-"],
pre[class*="language-"] {
	font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
}
```

The tag `kbd` uses the font `monospace` by default. If you have only `monospace` installed and none of the alternatives like `Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono',` it will work on your system.